### DISCUSSION TITLE
forwarding rules support DOH|DOT

### DISCUSSION THREAD

**Discussion by qcxiang** [DNSCrypt/dnscrypt-proxy#2398] [Ideas]
目前forwarding rules只能配置IP，没找到配置DOH或者DOT的方式。
dnscrypt-proxy目前不支持这个配置方式吗？还是我没找到方法。

**Comment by lifenjoiner** [comment]
No, there isn't, AFAIK.
If you really want to, relay them with extra proxies that support those protocols.

**Comment by ynyyn** [comment]
#1356 #1400 for reference.
